assets/scss/components/_popover.scss

```diff
@@ -33,6 +33,10 @@ $view-minWidth: 320px;
 
 .popover-container--menu {
 	min-width: $block-4;
+
+	&.visibility--a11yShow {
+		position:absolute;
+	}
```

Comment:
without this, the menu was `position:static` and took up more width than it should.